test/unit/bs3/module/Buttons.spec.js

```diff
@@ -99,6 +152,37 @@ define([
       });
     });
 
+    describe('strikethrough button state updated', function () {
+      it('should look toggled immediately when clicked', function () {
+        var $button = $toolbar.find('.note-btn-strikethrough');
+        assert.isTrue($button.length === 1);
+        assert.isFalse($button.hasClass('active'));
+        $button.click();
+        assert.isTrue($button.hasClass('active'), 'button is now active');
+      });
+    });
+
+    describe('clear button state not updated when clicked', function () {
+      it('should never look toggled when clicked', function () {
+        var $button = $toolbar.find('i.note-icon-eraser').parent();
+        assert.isTrue($button.length === 1);
+        assert.isFalse($button.hasClass('active'));
+        $button.click();
+        assert.isFalse($button.hasClass('active'), 'button is now active');
+      });
+    });
+
+    describe('font family button', function () {
+      it('should select the right font family name in the dropdown list when it is clicked', function () {
+        var $li = $toolbar.find('.dropdown-fontname li>a[data-value="Comic Sans MS"]');
+        var $span = $toolbar.find('span.note-current-fontname');
+        assert.isTrue($li.length === 1);
```

Comment:
One of these asserts is failing the current Travis build it seems, please check the assertions individually and this should fix the travis build and allow integration tests to pass.